projects/The Convo Space.md

```diff
@@ -0,0 +1,68 @@
+## Project Name
+The Convo Space
+
+## Category
+Protocol, Application
+
+## Project Description
+The Decentralized Conversation Layer of Internet, customizable plugins for apps connecting conversations and communities across the Internet.
+
+**Highlight features:**
+
+- Introducing an Open-Source Decentralized Conversation Layer that can be used to start instant conversations on any website/mobile-app/NFT/topic in a decentralized and unstoppable manner. 
+- Convo is an extensible layer that anyone can utilize to build conversational apps powered by Convo's API.
+- With Convo users can truly own their data and shows an aggregated view of all the conversations you've had powered by Convo's API throughout the internet on the Dashboard with the ability to delete them.
+- With Convo you can download your all of your own data and erase all of it from every website.
+- With Convo you can monetize your data by creating DataTokens and exposing your Data to the world of DeFi.
+- With Convo you can build you own interface/apps using the API to access user data (only by authentication).
+- With Convo you can further build and verify your decentralized digital identity with support for ENS along with Proof of Humanity and BrightID.
+
+## Use of IPFS, Filecoin and Libp2p
+The project utilizes Textile's ThreadDB and IPFS for building the Convo's API and Filecoin for data storage and backups.
+
+## Project Status
+The Beta was built during the Web3Weekend hackthon from ETHGlobal and won prizes from Protocol Labs, Textile, Ocean Protcol, ENS. 
+It's still under heavy development and would use the funds to enhance the existing features and buidling new ones.
+
+## Previews
+
+| Project Demo | Dashboard |
+|------|-------|
+|[![alt text](https://theconvo.space/images/poster.png)](https://youtu.be/p5-jQy7vKk8 "The Convo Space")|<img src="https://theconvo.space/images/dashboard.PNG">|
+
+## Target Audience
+Developers and Applications/Websites/MetaVerse
+
+## Rough estimated user base (if applicable)
+Early to say but the potential userbase is in ~Millions
+
+## Github repo
+https://github.com/anudit/convo
```

Comment:
Hi, these repos appear to be closed

Reply:
Ah, there you go @autonome

Reply:
great thanks! Can you apply MIT license? it is required for the grants program.

Reply:
Done @autonome